projects/components/src/filtering/filter-bar/filter-bar.component.ts

```diff
@@ -83,14 +84,16 @@ export class FilterBarComponent implements OnChanges, OnInit, OnDestroy {
   @Output()
   public readonly filtersChange: EventEmitter<Filter[]> = new EventEmitter();
 
-  @ViewChild('filterInput', { read: ElementRef })
-  public readonly filterInput!: ElementRef;
+  @ViewChild('filterInput')
+  public readonly filterInput?: FilterChipComponent;
 
   public isFocused: boolean = false;
 
   private readonly attributeSubject$: BehaviorSubject<FilterAttribute[]> = new BehaviorSubject<FilterAttribute[]>([]);
   private readonly internalFiltersSubject$: BehaviorSubject<Filter[]> = new BehaviorSubject<Filter[]>([]);
-  public readonly internalFilters$: Observable<Filter[]> = this.internalFiltersSubject$.asObservable();
+  public readonly internalFilters$: Observable<Filter[]> = this.internalFiltersSubject$
+    .asObservable()
+    .pipe(distinctUntilChanged(isEqual));
```

Comment:
Optimization, but sometimes multiple things (such as a url change and a child output) cause a filter change to be picked up, and we want to dedupe.